saleor/graphql/order/types.py

```diff
@@ -1015,9 +1106,16 @@ def _resolve_payment_status(payments):
     @staticmethod
     def resolve_payment_status_display(root: models.Order, info):
         def _resolve_payment_status(payments):
-            if last_payment := get_last_payment(payments):
+            last_payment = get_last_payment(payments)
+            # FIXME these conditions will be changed in separate PR after converting
+            #  current payment model into LegacyPayment object
+            if not last_payment:
+                return dict(ChargeStatus.CHOICES).get(ChargeStatus.NOT_CHARGED)
```

Comment:
Wouldn't be better to return this choice inline?

Reply:
`ChargeStatus.CHOICES` is a list of (str, str). I need to map it to dict, to get a readable description of status. Not sure if I can change it. I could only replace it with `str` but not sure if this woul be a best way for that